peerdid/peer_did.py

```diff
@@ -62,7 +104,7 @@ def save_peer_did(peer_did: PEER_DID, storage: Optional[Storage] = None):
     :raise ValueError: raises if peer_did parameter does not match peer_did spec
     """
     if is_peer_did(peer_did=peer_did):
```

Comment:
Please avoid such long indentation. 
It would be much cleaner to write
```
if not is_peer_did(peer_did):
    raise ValueError('Wrong Peer DID')
....
```